### PR TITLE
[1.13] Updates get_package_commands to check plugin.toml.

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

**Comment by armandgrillet** [comment]
@mesosphere-ci run all integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run mac integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run mac integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run all integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run all integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run mac integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run all integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run mac integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run all integration tests

File: ci/integration-tests-windows.groovy [hunk 1]
```diff
@@ -27,6 +27,8 @@ node('mesos-ubuntu') {
             checkout scm
             sh 'make windows'
             stash includes: 'build/windows/**', name: 'dcos-windows'
+            sh 'wget https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe'
```
Comment (bamarni) [inline ci/integration-tests-windows.groovy:30]: It'd be nice to have a comment indicating why we're doing this. For anyone who is not aware of the current HTTPS issues in our windows node it wouldn't be clear why we are stashing from here.

Comment (armandgrillet) [reply ci/integration-tests-windows.groovy:30]: OK, adding that as a comment in 1.12/1.13/14 branches in a commit on top later today.

**Pull request merged**